packages/client/yggdrasil/yggdrasil.spec

```diff
@@ -44,46 +44,31 @@ a receiving queue for instructions to be sent to the system via a broker.
 %global buildflags %{expand:-compiler gc -buildmode pie -tags=\\"rpm_crashtraceback libtrust_openssl\\" -ldflags \\"%ldflags\\" -a -v -x %{?**}}
 
 %build
-CGO_CPPFLAGS="-D_FORTIFY_SOURCE=2 -fstack-protector-all"  \
-BUILDFLAGS="%buildflags" \
-make PREFIX=%{_prefix} \
-     SYSCONFDIR=%{_sysconfdir} \
-     LOCALSTATEDIR=%{_localstatedir} \
-     LIBEXECDIR=%{_libexecdir} \
-     SHORTNAME=%{name} \
-     LONGNAME=%{name} \
-     PKGNAME=%{name} \
-     VERSION=%{version}
+%meson "-Dgobuildflags=%buildflags" %ldflags
```

Comment:
Is it really correct to pass the ldflags to meson? Looks like all versions don't like that. https://docs.fedoraproject.org/en-US/packaging-guidelines/Meson/ suggests that normally it should be able to figure it out by default.